templates/php-component/src/ConfigDefinition.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyComponent;
+
+use Keboola\Component\Config\BaseConfigDefinition;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+
+class ConfigDefinition extends BaseConfigDefinition
+{
+    protected function getParametersDefinition(): ArrayNodeDefinition
+    {
+        $parametersNode = parent::getParametersDefinition();
+        // @formatter:off
+        /** @noinspection NullPointerExceptionInspection */
```

Comment:
@tomasfejfar k cemu je tohle ?

Reply:
Vypíná v PhpStormu inspekci na null pointer. To "end()" může vracet null. Takže jakmile po něm něco zavoláš, tak je to error v PhpStormu

Reply:
aha, nodeDefinition->end() ma null, ale nodeBuilder->end() (kam jsem se dival) ne, jasne